UPGRADE-1.13.md

```diff
@@ -118,3 +118,5 @@
 1. Due to optimizations of the Order's grid the `Sylius\Component\Core\Repository\OrderRepositoryInterface::createSearchListQueryBuilder` method bas been deprecated and replaced by `Sylius\Component\Core\Repository\OrderRepositoryInterface::createCriteriaAwareSearchListQueryBuilder`.
    Also `Sylius\Component\Core\Repository\OrderRepositoryInterface::createByCustomerIdQueryBuilder` has been deprecated and replaced by `Sylius\Component\Core\Repository\OrderRepositoryInterface::createByCustomerIdCriteriaAwareQueryBuilder` for the same reason. Both changes affect
    `sylius_admin_order` and `sylius_admin_customer_order` grids configuration.
+
+1. The `Sylius\Bundle\ReviewBundle\EventListener\ReviewChangeListener` service is no longer listening for `postPersist`, `postUpdate`, and `postRemove` events. Instead, it listens for `postFlush` event, and then runs the subject's rating recalculation.
```

Comment:
This note seems not to be valid anymore and as we discussed internally, the PR should be opened to 1.12 🖖🏻